### ISSUE TITLE
pipenv uninstall leaves entry in Pipfile [packages]

### ISSUE THREAD

**Issue by tomu123** [pypa/pipenv#4863]
### Issue description

pipenv doesn't delete google.cloud package in pipfile or pipfile.lock after uninstalling it. I had to manually delete google.cloud entry from pipfile file in my project and then run `pipfile clean` to update my pipfile.lock file.

### Expected result

I expected to have pipfile and pipfile.lock files modified after uninstalling google.cloud package using `pipenv uninstall` command.

### Actual result

`pipenv uninstall google.cloud` only uninstalled the package but didn't modify the pipfile and pipfile.lock files from my project.

### Steps to replicate

Provide the steps to replicate (which usually at least includes the commands and the Pipfile).

```
pipenv install google.cloud
pipenv uninstall google.cloud
```

-------------------------------------------------------------------------------

Please run `$ pipenv --support`, and paste the results here. Don't put backticks (`` ` ``) around it! The output already contains Markdown formatting.

`pipenv --support|xclip 
Error: no such option: --support
`

**Comment by frostming** [comment]
I suspect you are using an ancient version of pipenv.

**Comment by tomu123** [comment]
I have downloaded pipenv from ubuntu repositories using `sudo apt install pipenv` following the steps from this repository.

**Comment by frostming** [comment]
Oh, the pipenv in apt repository is far behind the latest release, please use another installation method.

**Comment by tomu123** [comment]
Yeah you are right, I uninstalled it using `sudo apt purge pipenv` and then I installed it again using `pip install --user pipenv `. After that, I installed google.cloud package again and then uninstalled it, and pipenv updated my pipfile and pipfile.lock correctly.
Thank you so much for your help, I just hope They change the readme file installation recommendations since Ubuntu repository version of pipenv is outdated and pip works just fine without problem cross-platform. I proceed to close the issue.
